### PR TITLE
Added Products Listing Page

### PR DESCRIPTION
**Pull request by atharvabhanage02**
- Added the Product Listing Page 
- And Sidebar as Filter by Price , Category and Ratings.

Here is the Preview Link for the Product Listing Page:
https://deploy-preview-1--cricstore.netlify.app/

### CODE REVIEW

**Review by abhijeetekad — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+                    <span class="badge-count badge-count-right badge-circle-blue">4</span>
+                </div>
+                <div class="badges">
+                    <a href="#"><div class="icon wishlist-icon fas fa-shopping-cart"></div></a> 
+                    <span class="badge-count badge-count-right badge-circle-orange">4</span>
+                </div>
+            </div>
+        </div>
+    </nav>
+    <!-- Sidebar -->
```
Comment (ShreyashMahajan) [inline index.html:37]: you can use 'aside' for sidebar

**Review by ShreyashMahajan — APPROVED** [review]
Approved

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,124 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato&display=swap');
+body{
+    font-family: 'Lato', sans-serif;
+}
+*{
+    margin: 0;
+}
```
Comment (tusharkandpal) [inline styles.css:7]: can also add `box-sizing: border-box;` & `padding: 0;` here itself if needed

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+    z-index: 1;
+}
+.ecom-name{
+    text-decoration: none;
+    font-size: 1.8rem;
+    font-weight: bolder;
+    color: white;
+}
+.search-bar{
+    /* padding: 10px; */
```
Comment (tusharkandpal) [inline styles.css]: remove this comment please

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+    width: 100%;
+    outline: none;
+    border: none;
+}
+.search-container{
+    width: 20%;
+    display: flex;
+    /* margin: 30px 10px; */
```
Comment (tusharkandpal) [inline styles.css]: this too.

**Review by tusharkandpal — APPROVED** [review]
looks good ..! just go through points suggested below

File: styles.css (hunk 3 above)
Comment (atharvabhanage02) [reply styles.css]: Ohh Yes ! I'll remove those unwanted comments.

File: index.html (hunk 1 above)
Comment (atharvabhanage02) [reply index.html:37]: Yes sure . That would also be semantically more correct .

**Pull request merged**